### ISSUE TITLE
MultiActionButton support element type on text prop

### ISSUE THREAD

**Issue by StylianosKoidakisSage** [Sage/carbon#6720]
### Description

Same as Button component we should be able to pass an element to the text prop on the MultiActionButton, like so we can place some loaders or icons there

### Suggested solution

update the types of MultiActionButton

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nineteen88** [comment]
@harpalsingh This doesn't appear to be something actively supported in the DS. Do you think it is something we should support? 

@StylianosKoidakisSage Would it be possible to message Harpal and me on Slack with your use case for this? I assume it has come from a UX Designer who wishes to include an icon as part of the button?

**Comment by harpalsingh** [comment]
@nineteen88 taking a look at this and will speak with @ljemmo as well

**Comment by Parsium** [comment]
Hi @ljemmo @harpalsingh, were you both able to reach a conclusion on this? :)

**Comment by ljemmo** [comment]
@Parsium on my end, i would expect the main trigger on the multi action button to turn into a loading state when required. is this what is proposed? Currently can't see a visual so its hard to know what exactly is requested

**Comment by Parsium** [comment]
Hi @StylianosKoidakisSage 👋🏼 Apologies for the delay. Do you have any designs for using loaders within a multi-action button?

**Comment by edleeks87** [comment]
Yep @ljemmo is a loading state in the main button that's requested here, I'm going to raise a ticket and block it for now until we get some designs

**Comment by edleeks87** [comment]
FE-6770

**Comment by tomdavies73** [comment]
Hi @StylianosKoidakisSage, we've now made `SplitButton` and `MultiActionButton` accept a `forwardRef` and also utilised an imperative handle to surface specific focus methods. [Stackblitz here](https://stackblitz.com/edit/parsium-carbon-starter-escpsyzb?file=src%2FApp.tsx). 

We've opted for this approach so you can conditionally render a `Button` with a relevant loading state, then focus back to the `MultIActionButton`. Giving a seamless loading experience without the need for supporting specific loading capability and adding more complexity to `MultiActionButton`.

Now that we've provided an alternative pattern I'm going to close this issue, please feel free to get back in touch if you have any other questions 👍
